Classes/Milestones/MilestonesViewController.swift

```diff
@@ -55,6 +59,27 @@ MilestoneSectionControllerDelegate {
         super.viewDidLoad()
         navigationController?.navigationBar.titleTextAttributes = [.foregroundColor: UIColor.white]
         addMenuDoneButton()
+        addMenuClearButton()
+    }
+
+    func addMenuClearButton() {
+        navigationItem.leftBarButtonItem = UIBarButtonItem(
+            title: NSLocalizedString("Clear", comment: ""),
```

Comment:
Can we add this string to the `Constants.Strings` list since its used multiple times here?